light-client/tests/support/model_based/single_step/MC4_4_faulty_TestHeaderFromFuture.json

```diff
@@ -25,70 +25,73 @@
         "height": "1",
         "round": 1,
         "block_id": {
-          "hash": "B33760C2EF89CA5535437C9E04DC87A0A1CB279E15FE3554171565AD04CD4F9B",
-          "parts": null
+          "hash": "765CAC565626BF7405C8B07ECFC06BF202F969A1F1066897BEFD09A1866BF588",
+          "parts": {
+            "total": 0,
+            "hash": "0000000000000000000000000000000000000000000000000000000000000000"
+          }
```

Comment:
This is why the tests had to be recreated.

Reply:
I see:) yes, there are a lot of yet incomplete pieces in the Testgen; we will fix them as soon as something breaks.

I think I will slightly tweak the model-based tests execution, such that the files will be created with the needed names already -- this will make the regeneration process a bit easier, until we change or automated this.